tests/test_wave_01.py

```diff
@@ -119,12 +119,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
```

Comment:
As a minor style preference, I recommend just getting rid of commented out code in general unless there's a clear reason to keep it, which should be explained in a comment. This also applies to the comments below where we described how we wanted you to extend the tests, but more just in the sense of keeping code clean.

Deleting these comments would have the additional benefit of keeping the `assert` on line 126 closer to the other asserts in this function.

I'm sure there's a few other cases where this applies, but I'll only point it out once to avoid repeating myself.